tests/conftest.py

```diff
@@ -218,6 +218,20 @@ def random_grains(grains_config: dict, random_filters: Filters, tmp_path) -> Gra
     return grains
 
 
+@pytest.fixture
+def small_array_filters(small_array: np.ndarray, filter_config: dict, tmp_path) -> Grains:
+    """Grains object based on small_array."""
```

Comment:
This is returning a `Filters` object not a `Grains`.

```suggestion
    """Filters object based on small_array."""
```